pkg/arm/v5/resources.go

```diff
@@ -213,14 +217,85 @@ func (g *simpleGenerator) lbKubernetes() *network.LoadBalancer {
 							)),
 						},
 					},
-					Name: to.StringPtr(lbKubernetesOutboundFrontendConfigurationName),
 				},
 			},
 			BackendAddressPools: &[]network.BackendAddressPool{
 				{
 					Name: to.StringPtr(lbKubernetesBackendPoolName),
 				},
 			},
+			LoadBalancingRules: &[]network.LoadBalancingRule{
+				{
+					LoadBalancingRulePropertiesFormat: &network.LoadBalancingRulePropertiesFormat{
+						FrontendIPConfiguration: &network.SubResource{
+							ID: to.StringPtr(resourceid.ResourceID(
+								g.cs.Properties.AzProfile.SubscriptionID,
+								g.cs.Properties.AzProfile.ResourceGroup,
+								"Microsoft.Network/loadBalancers",
+								lbKubernetesName,
+							) + "/frontendIPConfigurations/" + lbKubernetesOutboundFrontendConfigurationName),
+						},
+						BackendAddressPool: &network.SubResource{
+							ID: to.StringPtr(resourceid.ResourceID(
+								g.cs.Properties.AzProfile.SubscriptionID,
+								g.cs.Properties.AzProfile.ResourceGroup,
+								"Microsoft.Network/loadBalancers",
+								lbKubernetesName,
+							) + "/backendAddressPools/" + lbKubernetesBackendPoolName),
+						},
+						Probe: &network.SubResource{
+							ID: to.StringPtr(resourceid.ResourceID(
+								g.cs.Properties.AzProfile.SubscriptionID,
+								g.cs.Properties.AzProfile.ResourceGroup,
+								"Microsoft.Network/loadBalancers",
+								lbKubernetesName,
+							) + "/probes/" + lbKubernetesProbeName443),
+						},
+						Protocol:            network.TransportProtocolTCP,
+						LoadDistribution:    network.Default,
+						FrontendPort:        to.Int32Ptr(443),
+						BackendPort:         to.Int32Ptr(443),
+						EnableFloatingIP:    to.BoolPtr(false),
+						DisableOutboundSnat: to.BoolPtr(true),
```

Comment:
why this line?

Reply:
add IdleTimeoutInMinutes to match the others

Reply:
The disableSnat was because when using the outbound it must be disabled.  Since we are creating a new one I will remove this option.

Also set timeout equal to others.